.github/workflows/deploy.yml

```diff
@@ -4,6 +4,7 @@ on: push
 
 jobs:
   deploy:
+    if: github.repository_owner == 'TerriaJS'
```

Comment:
Doesn't this need to be `if: ${{ github.repository_owner == 'TerriaJS' }}` like the example at https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#determining-when-to-use-contexts?